gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -50,4 +55,20 @@ else
     trap 'docker kill {genconf_tar}' HUP QUIT INT TERM
     docker run --rm --name=$DCOS_INSTALLER_CONTAINER_NAME -i -p $PORT:9000 -v $(pwd)/genconf/:/genconf {docker_image_name} "$@"
 fi
-exit $?
+
+if [ $? -ne 0 ]; then
```

Comment:
We have `errexit` aka `-e` set, so this should not be necessary. At this point we `$?` must be 0.

Reply:
Ah, I wonder why the original script was using exit $? .. since it would always have to be 0 if docker exited cleanly.